### PR TITLE
[Tests-Only] Run GUI tests on nightly

### PR DESCRIPTION
**Pull request by swoichha**
### Description
Currently we do not run GUI tests on nightly. So, this PR will run all the GUI tests against the `daily-master-qa` server and  run PRs against a stable server tarball `10.7.0`.

After this PR gets merged we need to change the server version from `10.7.0` to latest stable release when new stable version of core is released.

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -50,8 +55,8 @@ def main(ctx):
             "Ninja",
             trigger = build_trigger,
         ),
-        gui_tests(ctx, trigger = build_trigger, filterTags = ["@smokeTest"]),
-        gui_tests(ctx, trigger = build_trigger, depends_on = ["GUI-tests-@smokeTest"], filterTags = ["~@smokeTest", "~@skip"]),
+        gui_tests(ctx, trigger = build_trigger, filterTags = ["@smokeTest"], version = "10.7.0"),
```
Comment (phil-davis) [inline .drone.star:58]:
```suggestion
        gui_tests(ctx, trigger = build_trigger, filterTags = ["@smokeTest"], version = "latest"),
```

File: .drone.star [hunk 2]
```diff
@@ -50,8 +55,8 @@ def main(ctx):
...
+        gui_tests(ctx, trigger = build_trigger, depends_on = ["GUI-tests-@smokeTest"], filterTags = ["~@smokeTest"], version = "10.7.0"),
```
Comment (phil-davis) [inline .drone.star:59]:
```suggestion
        gui_tests(ctx, trigger = build_trigger, depends_on = ["GUI-tests-@smokeTest"], filterTags = ["~@smokeTest"], version = "latest"),
```

**Review by phil-davis — COMMENTED** [review]
We can mention "latest" and that will install the latest release. The download server has links for "latest" that always redirect to the latest core server release tarball. Then we do not have to remember to update `.drone.star` here. The disadvantage is that on the day of official release, the CI might start failing here (but we will have been doing PRs against daily-master-qa anyway - so we should know of any upcoming problems with a server release, and stop server team from releasing it!)

File: .drone.star (hunk 1 above)
Comment (swoichha) [reply .drone.star]: Fixed

File: .drone.star (hunk 2 above)
Comment (swoichha) [reply .drone.star]: Fixed

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/client/7530/6/6
```
$ wget -qO- https://download.owncloud.org/community/daily/owncloud-latest.tar.bz2 | tar -xj -C /tmp/owncloud/ --strip 1
Fetched 10.7.0 - channel: stable - build at 2021-03-26T15:55:58+00:00 - SHA: 119742ed5f9c01fd8fdae86457fd573533063a83
```

Good - that is installing the "latest" server core release 10.7.0 during CI for the PR.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/client/settings has a nightly "cron" - so we can check tomorrow for how that job ran.

**Comment by phil-davis** [comment]
@TheOneRing `test-clang-format` fails CI. This PR did not touch any C*-code.
Is that a known CI fail?

I will merge this PR anyway.

**Comment by TheOneRing** [comment]
Yes looks like the github action broke over night

**Pull request merged**